t/page/house.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+require 'test_helper'
+require_relative '../../lib/page/house'
+
+describe 'House' do
+  subject do
+    country = index_at_known_sha.country('united-states-of-america')
+    house   = country.legislature('senate')
+    Page::House.new(country: country, house: house)
+  end
+
+  it 'has the country' do
+    subject.country.name.must_equal 'United States of America'
+  end
+
+  it 'has the house' do
+    subject.house.name.must_equal 'Senate'
+  end
+
+  describe 'legislative periods' do
+    it 'has all legislative periods' do
+      subject.legislative_periods.count.must_equal 18
+    end
+
+    it 'is a list of terms' do
+      subject.legislative_periods.first.slug.must_equal '114'
+    end
```

Comment:
The test here doesn't seem to match the description. I'm not sure what you're actually trying to test, but "The first things in this list must have a slug of 114" doesn't seem to be a good way to check that you have a list of terms…